### ISSUE TITLE
Columns re-order

### ISSUE THREAD

**Issue by zeysh** [norwoodj/helm-docs#73]
Hello,

Question:
I wonder whether there is a way to change the order in the columns or not ?
I would like to switch `description` and `default value` columns.

Many thanks

**Comment by norwoodj** [comment]
I don't think that this should be configurable, however, I think that the ordering of columns that you propose is a better one. I will change it.

**Comment by cmoulliard** [comment]
As most of the existing helm documented [projects](https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/VALUES_SUMMARY.md) proposes as default, by convention, the following columns: Parameter | Description |  Default, then I think that it should be great that we can configure the columns generated like also their order
